.github/workflows/main.yml

```diff
@@ -23,7 +23,7 @@ jobs:
             _ver="['3.8', '3.9', '3.10']"
             _cache="1"
           else 
-            _ver="['3.8']"
+            _ver="[3.8]"
```

Comment:
can you put a comment above it that it should stay that way so the `arc` scenario works?